### PR TITLE
fix(aio): accessibility items

### PR DESCRIPTION
**Pull request by sjtrimble**
https://github.com/angular/angular/issues/16005

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe:
https://github.com/angular/angular/issues/16005

**What is the current behavior?** (You can also link to an open issue here)



**What is the new behavior?**



**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...


**Other information**:

### CODE REVIEW

**Comment by IgorMinar** [comment]
Please rebase this PR

File: aio/src/styles/1-layouts/_layout-global.scss [hunk 1]
```diff
@@ -34,4 +34,8 @@ l-relative {
 
 .center {
     text-align: center;
+}
+
+.a-hidden {
+    display: none;
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_layout-global.scss]: I think that `display:none` is usually not announced by screen readers. So, this won't work as intended.

**Review by gkalpak — CHANGES_REQUESTED** [review]
One minor issue. LGTM othrwise.

**Comment by sjtrimble** [comment]
@gkalpak I changed the way the hidden class works so it should only apply to the visual aspect now. Thanks for pointing out!

File: aio/src/styles/1-layouts/_layout-global.scss [hunk 2]
```diff
@@ -34,4 +34,10 @@ l-relative {
 
 .center {
     text-align: center;
+}
+
+.hidden {
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_layout-global.scss]: Nit: I think it is common to use a name like `.visually-hidden` to convey that it is not hidden from screen-readers.

**Review by gkalpak — APPROVED** [review]
LGTM (1 minor nit - not a blocker)

File: aio/src/styles/1-layouts/_layout-global.scss (hunk 2 above)
Comment (sjtrimble) [inline aio/src/styles/1-layouts/_layout-global.scss]: Made the change. I agree.

**Review by gkalpak — APPROVED** [review]

**Pull request merged**